### DISCUSSION TITLE
3rd中的jemalloc在skynet编译成功之后可以clean掉吗

### DISCUSSION THREAD

**Discussion by shuitian** [cloudwu/skynet#1674] [Q&A]
现在编译skynet会自动编译jemalloc。会有很多中间文件*.o *.d 以及lib/libjemalloc.so.2 lib/libjemalloc.so lib/libjemalloc.a lib/libjemalloc_pic.a
这些文件在skynet运行时有用吗，在正式环境下可以删除吗

**Comment by hanxi** [comment]
简单的方法就是你删除试试能不能运行

**Reply by shuitian** [reply]
我试过了，可以运行，就是不知道有没有其他地方有调用到

**Comment by hanxi** [comment]
看makefile用的是静态库，编译到skynet可执行文件里了
